### PR TITLE
 chore: comments typo fix; make code intention more clearer and eliminate 1 line of duplicate

### PR DESCRIPTION
**Pull request by Jeff-Tian**
##### Checklist

- [x] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message follows commit guidelines

##### Affected core subsystem(s)
N.A.

##### Description of change
1. comments typo fix;
2. eliminate 1 line of duplicate code

### CODE REVIEW

File: lib/utils.js [hunk 1]
```diff
@@ -88,9 +88,7 @@ module.exports = {
       message = util.format.apply(util, args);
     }
 
-    if (meta.raw === true) {
-      output = message;
-    } else if (options.json === true || formatter) {
+    if ((options.json === true || formatter) && meta.raw !== true) {
```
Comment (dead-horse) [inline lib/utils.js]: what's the purpose of this change? If this is a bug can you add a test case for this change?

Comment (Jeff-Tian) [reply lib/utils.js]: 目的：消除一行代码重复；不是 BUG，没有实质性影响，就是阅读代码时，看到 if 和 else 分支重复了，感觉不是很自然，合并了一下，代码意图更清晰一些。没有加新的测试，原有测试都通过了。

Comment (dead-horse) [reply lib/utils.js]: 这样导致一个 if 条件中分支过多，意图更加不清晰，还是按照原来的吧

Comment (Jeff-Tian) [reply lib/utils.js]: OK
